src/governance.rst

```diff
@@ -144,6 +148,25 @@ the time when a vote is called, but may never be less than 24 hrs.
     * Sensitive
     * 75% Majority to pass
 
+:Overall workflow and packaging policies: The proposer can choose to
+    create a poll with an external tool or call
+    for voting on the GH issue in question.
+    The voting must period be open for at least one core
+    member meeting cycle to allow for clarification questions
+    and discussions. Friendly reminders to vote are encouraged.
+
+    * Standard
+    * 50% plus one of those voting to pass
+
+:Modifying the governance document: The voting should happen in the PR
+    in question and there must be a call to `@conda-forge/core`.
+    The voting period must be open for at least one core
+    member meeting cycle to allow for clarification questions
+    and discussions.
+
+    * Standard
+    * 75% of those voting to pass
```

Comment:
Should this be "75% plus one" or "More than 75%" or "75% or more"? Don't have strong feelings about any of those options, but if the intent is that "3 out of 4 votes are yes ⇒ pass" then we should go with the last of those options.....